src/common/blacklist.js

```diff
@@ -1,4 +1,9 @@
-const blacklist = ["renovate-bot", "technote-space", "sw-yx"];
+const blacklist = [
+  "renovate-bot",
+  "technote-space",
+  "sw-yx",
+  "blacklistedUser", // For testing purposes
+];
```

Comment:
Is this a problem? Makes testing easier.

Reply:
Should not be a problem.